walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java

```diff
@@ -21,15 +21,27 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns HTML that contains the page view count. */
-@WebServlet("/page-views")
+// @webservlet is an annotation that tells our server which UTL this ervlet maps to
+//when client requests the pageviews URL, this servlet is triggered
+//@WebServlet("/content")
+
+@WebServlet("/page-views") 
 public class PageViewServlet extends HttpServlet {
 
   private int pageViews = 0;
-
+  //do get function below runs whenever a client sends a GET request to the servlet URL
+  //aka browser seds a GET request whenever I visit a URL
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     pageViews++;
+    //^ since the doget function runs everytime the pageviews URL is requested, 
+    //this means the pageviews variable tracks how many times the page has been viewed
 
+    /*
```

Comment:
You many want to remove this comment? As the following snippet is quite similar.